db/Pos/DB/Class.hs

```diff
@@ -1,24 +1,67 @@
-{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE Rank2Types          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies        #-}
+{-# LANGUAGE TypeOperators       #-}
 
--- | Class which provides access to database.
+-- | A set of type classes which provide access to database.
+--
+-- 'MonadDB' is the most featured class (actually just a set of
+-- constraints) which wraps 'NodeDBs' which contains RocksDB
+-- databases. This class can be used to manipulate RocksDB
+-- directly. It may be useful when you need an access to advanced
+-- features of RocksDB.
+--
+-- Apart from that we have three more classes here.
+--
+-- 'MonadDBPure' contains only 'dbGet' method.  The advantage of it is
```

Comment:
Doesn't say anything about the fact it's the main class for DB emulation. And I don't like the naming (already told it in slack).